lib/internal/Magento/Framework/View/Layout/CacheKeyInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\View\Layout;
+
+/**
+ * Interface CacheKeyInterface
+ */
+interface CacheKeyInterface
+{
+    /**
+     * Add cache key for generating different cache id for same handles
+     *
+     * @param array|string $cacheKey
+     */
```

Comment:
return type is missing

Reply:
This method returns nothing, should it be noted as @return void?

Reply:
Yes, docBlock should be clearly informing on that